File: FuncamentosProgramacao/CapituloTres/AtividadeNove.cs
namespace FuncamentosProgramacao.CapituloTres;

public class AtividadeNove
{
    public static void CalculaAreaTrapezio()
    {
        double baseMaior;
        double baseMenor;
        double altura;
        double areaTrapezio;

        Console.Write("Informe o valor da base maior: ");
        baseMaior = double.Parse(Console.ReadLine()!);
        Console.Write("Informe o valor da base menor: ");
        baseMenor = double.Parse(Console.ReadLine()!);
        Console.Write("Informe o valor da altura: ");
        altura = double.Parse(Console.ReadLine()!);

        areaTrapezio = ((baseMaior + baseMenor) * altura) / 2;

        Console.WriteLine($"A área do trapézio de base maior {baseMaior}m, base menor {baseMenor}m e altura {altura}m " +
                          $"é: {areaTrapezio:F}m²");
    }
}

File: fundamentosProgramacaoJava/src/capitulo/tres/AtividadeNove.java
package capitulo.tres;

import java.util.Scanner;

public class AtividadeNove {

    public static void main(String[] args) {

        float baseMenor;
        float baseMaior;
        float altura;
        float area;
        Scanner leia = new Scanner(System.in);


        System.out.print("Digite o valor da base menor: ");
        baseMenor = leia.nextFloat();
        System.out.print("Digite o valor da base maior: ");
        baseMaior = leia.nextFloat();
        System.out.print("Digite o valor da altura: ");
        altura = leia.nextFloat();

        area = ((baseMaior + baseMenor) * altura) / 2;

        System.out.print("O valor da area do trapezio e: " + area + "m²");

        leia.close();

    }

}
